nautobot/docs/development/apps/api/ui-extensions/object-views.md

```diff
@@ -2,22 +2,25 @@
 
 ## Extending Object Detail Views
 
-Apps can inject custom content into certain areas of the detail and list views of applicable models. This is accomplished by subclassing `TemplateExtension`, designating a particular Nautobot model, and defining the desired attributes and/or methods to provide custom content. Two attributes and six methods are available:
+Apps can inject custom content into certain areas of the detail and list views of applicable models. This is accomplished by subclassing `TemplateExtension`, designating a particular Nautobot model, and defining the desired attributes and/or methods to provide custom content. Several attributes and methods are available:
 
 * `object_detail_tabs` - List of `Tab` instances to add to the detail view as additional tabs.
+* `object_detail_buttons` - List of `Button` instances to add to the detail view.
 * `object_detail_panels` - List of `Panel` instances to add to the main tab of the detail view.
 * `left_page()` - Inject content on the left side of the object detail page (deprecated since Nautobot 2.4.0; `object_detail_panels` is preferred)
 * `right_page()` - Inject content on the right side of the object detail page (deprecated since Nautobot 2.4.0; `object_detail_panels` is preferred)
 * `full_width_page()` - Inject content across the entire bottom of the object detail page (deprecated since Nautobot 2.4.0; `object_detail_panels` is preferred)
-* `buttons()` - Add buttons to the top of the object detail page
+* `buttons()` - Add buttons to the top of the object detail page (deprecrated since Nautobot 2.4.0; `object_detail_buttons` is preferred)
```

Comment:
```suggestion
* `buttons()` - Add buttons to the top of the object detail page (deprecated since Nautobot 2.4.0; `object_detail_buttons` is preferred)
```